queries/rust/highlights.scm

```diff
@@ -71,6 +71,9 @@
   name: (identifier) @namespace))
 (scoped_type_identifier
   path: (identifier) @namespace)
+(scoped_type_identifier
+  path: (identifier) @type
+  (#lua-match? @type "^[A-Z]"))
```

Comment:
These sort of "ALL CAPS is different" has historically not been very popular; too likely to trigger false positives in some contexts.

Reply:
Scroll up and down -- I'm just fixing an existing scheme of differentiation by capitalisation that forgot a case. For Rust it has not been a problem because the compiler has enabled-by-default warnings on types that aren't camel-case, and modules or functions that aren't snake-case.

Reply:
As long as it's deliberate and targeted for the language, that's fine by me.